src/ODEInterface.jl

```diff
@@ -161,11 +161,12 @@ end
   """
 const solverInfo = Vector{SolverInfo}() 
 
-
+# Work around syntax changes in v0.6 without requiring Compat
+eval(Expr(:abstract, :(AbstractArgumentsODESolver{FInt} <: Any)))
 """
   Ancestor for all types storing arguments for ODE-(C-/Fortran-)solvers.
-  """
-abstract AbstractArgumentsODESolver{FInt} <: Any
+"""
+AbstractArgumentODESolver
```

Comment:
If you have `abstract ...` in v0.5 or below, you can do `eval(Expr(:abstract, :( ... )))`
and it will work on any version.  It's a really easy transformation to make (except for having to deal with docstrings, but that was also broken when using `@compat` last time I tried it.